settings/global.mdx

```diff
@@ -249,7 +249,7 @@ Example:
   ```
   <Expandable title="FooterSocials">
     <ResponseField name="[key]" type="string">
-    One of the following values `website`, `facebook`, `twitter`, `discord`, `slack`, `github`, `linkedin`, `instagram`, `hacker-news`
+    One of the following values `website`, `facebook`, `twitter`, `x`, `youtube`, `discord`, `slack`, `github`, `linkedin`, `instagram`, `hacker-news`
     
     Example: `twitter`
```

Comment:
Let's change the example here cause `twitter` is not the preferred value. Maybe `linkedin`?

Reply:
do we want `"x": "https://twitter.com/mintlify"`?